site/docs/tools/index.md

```diff
@@ -4,16 +4,14 @@ Nessie is focused on working with the widest range of tools possible. If a tool
 or reads data, Nessie seeks to work with it. Current Nessie integrations/tools include 
 the following:
 
-- [Iceberg Integration](iceberg/index.md).
-    - [Spark](iceberg/spark.md).
-
-- [Delta Lake Integration](deltalake/index.md).
-    - [Spark](deltalake/spark.md).
-
-- [Nessie CLI](cli.md).
-- [Nessie Web UI](ui.md).        
+- [Iceberg Integration](iceberg/index.md)
```

Comment:
Has anything changed here? GH shows a diff, but I don't see any change?

Reply:
I just removed the `.` at the end of each line